src/translations/sv.json

```diff
@@ -79,6 +79,7 @@
   "page.berths.fence": "Port",
   "page.berths.lighting": "Belysning",
   "page.berths.storage_for_docking_equip": "Sommarförvaring för utrustning",
+  "page.berths.storage_for_trailers": "Jag behöver en tillgänglig plats",
```

Comment:
wait, what?

Reply:
label says "I need an accessible place"

Reply:
it should be [this text](https://github.com/City-of-Helsinki/berth-reservations-ui/blob/1575d9f75496e88bdbb7ec7a98e137a815b86de3/src/translations/sv.json#L254)

why are these in `page.berths.` btw?

Reply:
> label says "I need an accessible place"

This leaked from another task 😃 

> why are these in page.berths. btw?

The labels of the icons are still under `page.berths.` because when storage areas were added they shared the same berth page that was already implemented, later they were separated from each other.
But it's a good point, and since I touched this file I can move the winter storage labels.

Reply:
👍